src/state/reducer.ts

```diff
@@ -37,10 +37,21 @@ const appReducer = combineReducers({
 
 export type AppState = ReturnType<typeof appReducer>
 
+function createStorage(dbName: string) {
+  const db = localForage.createInstance({
+    name: dbName,
+  })
+  return {
+    getItem: db.getItem,
```

Comment:
This will not get old state, because it will look in indexedDB for it instead of localStorage. You'll need to write your own migration to pull it out of localStorage and into localForage.

Reply:
@vm the initial migration can be done in `migrations.ts` - currently i'm copying the old localStorage `lists` into the new naming scheme but leaving it in localStorage. you can modify that to remove it from localStorage entirely and instead write it to the indexDB